### PR TITLE
Pane for small screens

### PR DESCRIPTION
**Pull request by kagz**

### CODE REVIEW

File: src/components/index.jsx [hunk 1]
```diff
@@ -32,33 +34,68 @@ function Home() {
...
+	const [open, setOpen] = useState(false);
+
 	return (
 		<>
 			<div className="container">
 				<div className="main-body">
 					<h2>Search For Developer Jobs</h2>
 					<div className="main-card">
-						<div>
+						<div className={cn(' form-wrapper', open && 'active')}>
```
Comment (mpalmr) [inline src/components/index.jsx]: Remove prepending space on `form-wrapper`

File: src/components/index.jsx [hunk 2]
```diff
@@ -32,33 +34,68 @@ function Home() {
...
 									<Form.Control placeholder="Where?" type="text" name="location" className="custom-text" />
 								</Form.Group>
-
 								<Form.Group>
 									<Form.Label className="form-label">Type</Form.Label>
-									<Form.Control as="select" custom className="custom-text">
-										<option>Full Time</option>
-										<option>Part Time</option>
-									</Form.Control>
+									<div className="custom-text text-center">
```
Comment (mpalmr) [inline src/components/index.jsx]: `custom-text` is too generic of a class name

File: src/components/index.jsx (hunk 1 above)
Comment (mpalmr) [reply src/components/index.jsx]: I also feel like this should be its own component that includes the button that opens and closes it.

File: src/styles/index.scss [hunk 3]
```diff
@@ -16,28 +16,36 @@ color: #b7cbd9;
...
+}
 
+// custom select
+
+.custom-control-label::before {
+	background-color:#d97b16;
+	border: #b7cbd9 solid 1px;
 }
 
-.search-part form{
```
Comment (mpalmr) [inline src/styles/index.scss:39]: Isn't `.search-part` on the form? This requires it to be a child of `.search-part`.

File: src/styles/index.scss [hunk 4]
```diff
@@ -58,30 +66,29 @@ width: 100%;
...
 	justify-content: center;
 	padding-top: 8px;
 }
 
-
-
-.page-link, .page-item:first-child .page-link ,.page-item:last-child .page-link{
+.page-link,
+.page-item:first-child .page-link,
+.page-item:last-child .page-link {
```
Comment (mpalmr) [inline src/styles/index.scss:89]: What's the purpose of this? Can't it just be `.page-link`?

File: src/styles/index.scss [hunk 5]
```diff
@@ -114,56 +130,53 @@ margin: 0;
 	box-shadow: 5px 5px 10px rgba(27, 27, 27, 0.5);
 }
 
-.card-box>a{
+.card-box > a {
 	color: inherit;
-
 }
```
Comment (mpalmr) [inline src/styles/index.scss]: &:hover

File: src/styles/index.scss [hunk 6]
```diff
@@ -173,6 +186,53 @@ font-size: 20px;
...
 	font-size: 17px;
 }
-.view-btn:hover{
-	background: #097205;;
+.view-btn:hover {
+	background: #097205;
+}
+
+.form-wrapper {
+  @media (max-width: 768px) {
```
Comment (mpalmr) [inline src/styles/index.scss]: Put this in the initial `.form-wrapper` definition

**Review by mpalmr — CHANGES_REQUESTED** [review]

File: src/styles/index.scss (hunk 4 above)
Comment (kagz) [reply src/styles/index.scss:89]: i targeted the class when i was styling some paginations

File: src/components/index.jsx (hunk 2 above)
Comment (kagz) [reply src/components/index.jsx]: changed it to custom-checkbox

File: src/styles/index.scss (hunk 6 above)
Comment (kagz) [reply src/styles/index.scss]: what do yo  mean here

File: src/components/index.jsx (hunk 1 above)
Comment (kagz) [reply src/components/index.jsx]: i solved this

**Pull request closed without merging**